### ISSUE TITLE
Go Get fails with Missing type "EvalNode"

### ISSUE THREAD

**Issue by TGiFallen** [golang/lint#129]
``` go version ```
``` go1.4.2 windows/amd64 ```
The error I receive upon trying to install golint is 

``` golang\lint\lint.go:1483: undefined type: types.EvalNode ```

So it seems that go is refusing to download go types, even though it's mentioned in the import. I have tried to manually install it but I probably did that wrong. My GOPATH is set up correctly (I think, GOPATH = C:\Home\Programming\go , and it has the bin, pkg and src folders inside of it.)

Thanks in advance.

**Comment by TGiFallen** [comment]
Upon some further poking around, when i run ``` go get -u github.com/golang/lint/golint``` It downloads and compiles the go types package (shows up in %GOPATH%\pkg\windows_amd64\golang.org\x\tools\go\types.a) but it's still erroring, saying it's missing the evalnode type. does this mean it's not linking properly?

**Comment by markmandel** [comment]
I'm getting exactly the same issue as well.

```bash
➜  default git:(master) ✗ export GOPATH=`pwd`
➜  default git:(master) ✗ printenv | grep GO
GOPATH=/home/mark/workspace/gopath/default
➜  default git:(master) ✗ go get -u github.com/golang/lint/golint
# github.com/golang/lint
src/github.com/golang/lint/lint.go:1483: undefined: types.EvalNode
➜  default git:(master) ✗
```

It was working up to a few hours ago. I see there are several commits to - https://github.com/golang/tools/ ,  which could be causing the issues. Looks like it's being actively worked on right now.

**Comment by markmandel** [comment]
Yep, now I'm getting this error:

```bash
goapp get github.com/golang/lint/golint
# golang.org/x/tools/go/types
src/golang.org/x/tools/go/types/eval.go:68: undefined: parser.ParseExprFrom
```

**Comment by dsymonds** [comment]
Yep, @griesemer broke the x/tools/go/types for stable versions of Go. He's rolling back his change.

**Comment by dsymonds** [comment]
Rollback in https://go-review.googlesource.com/#/c/11010/; run `go get -u github.com/golang/lint` and you should be fine.

**Comment by TGiFallen** [comment]
I deleted the pkg folder and src/github.com/golang (refresh it, just in case) And I'm still getting the same error as before (missing type, EvalNode) Is there anything else i should be doing?

**Comment by griesemer** [comment]
Ah! EvalNode is gone... - this needs a bit more work.

**Comment by TGiFallen** [comment]
@griesemer haha, thanks for fast work on fixing this.

**Comment by dsymonds** [comment]
I believe it's really fixed now (at b0bb8ee).

**Comment by TGiFallen** [comment]
Yes sir, it is, Thanks again!
